templates/post/_options.html.twig

```diff
@@ -1,94 +1,262 @@
-<aside class="options" id="options">
+{% set showFilterLabels = app.request.cookies.get('kbin_general_filter_labels')|default('on') %}
+
+<aside class="options options--top" id="options">
     <div></div>
-    <menu class="options__main">
-        <li>
-            <a href="{{ options_url('sortBy', 'top') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'top')}) }}">
-                {{ 'top'|trans }}
-            </a>
-        </li>
-        <li>
-            <a href="{{ options_url('sortBy', 'hot') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'hot')}) }}">
-                {{ 'hot'|trans }}
-            </a>
-        </li>
-        <li>
-            <a href="{{ options_url('sortBy', 'newest') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'newest')}) }}">
-                {{ 'newest'|trans }}
-            </a>
-        </li>
-        <li>
-            <a href="{{ options_url('sortBy', 'active') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'active')}) }}">
-                {{ 'active'|trans }}
-            </a>
-        </li>
-        <li>
-            <a href="{{ options_url('sortBy', 'commented') }}"
-               class="{{ html_classes({'active': route_has_param('sortBy', 'commented')}) }}">
-                {{ 'commented'|trans }}
-            </a>
+    <menu class="options__filter">
```

Comment:
same problem here, the class `options__main` needs to be added

Reply:
Looking at it, we might be fine as is as, and I could be wrong, I can't get the content to take up enough space to require a scroll bar

instead we could add

```js
         if (container) {
             const containerWidth = container.clientWidth;
             const area = container.querySelector('.options__main');
+
+            if (null === area) {
+                return;
+            }
```

which will stop the js error as it won't attempt to make the option bar a scrollbar unless it's set up with those classes (maybe we can name it better, I see @andrewmoise removed `no-scroll` class and it does sort of seem like it might've been unused, unless I was missing it, so maybe we could make a class specifically for when a scroll bar should be introduced

but I think we should make more things drop downs, the user profile which has a ton of options being one of them, which is already something KES had

Reply:
sure we can do that as well